src/RA_Integration.vcxproj

```diff
@@ -74,6 +74,7 @@
     <ClInclude Include="RA_Dlg_Memory.h" />
     <ClInclude Include="RA_Dlg_RichPresence.h" />
     <ClInclude Include="RA_Dlg_RomChecksum.h" />
+    <ClInclude Include="ra_fwd" />
```

Comment:
why do these new files not have an extension, and why is this one "ClInclude" whereas the other two are "None"?